vertical-pod-autoscaler/admission-controller/logic/server.go

```diff
@@ -64,23 +64,47 @@ func (s *AdmissionServer) getPatchesForPodResourceRequest(raw []byte, namespace
 	patches := []patchRecord{}
 	updatesAnnotation := []string{}
 	for i, containerResources := range containersResources {
```

Comment:
Will this mean that we set the objects for all pods or only those controlled by VPA?

Reply:
Only for containers those controlled by VPA. Note that 'containersResources' contains recommendations - i.e. we only iterate if there are recommended values.